client-js/app/scripts/app.ts

```diff
@@ -247,17 +257,102 @@ window.addEventListener('WebComponentsReady', () => {
     }
   }
 
+  class DatavaultLiveActivity implements Activity {
```

Comment:
I understand now why there are two of these, but there seems to be a lot of code duplicated between those two functions. It's beyond this PR, but I think it would be nice to have a lower (higher? I forget the convention) level function with the common skeleton called ```DataVaultActivity``` that ```DataVaultLiveActivity``` and something like ```DataVaultStaticActivity``` inherit from.